packages/lab/src/pill-next/PillNext.css

```diff
@@ -72,22 +61,22 @@
   color: var(--pillNext-text-color-active);
 }
 
-.saltPillNext-clickable.saltPillNext-disabled.saltPillNext-active,
-.saltPillNext-clickable.saltPillNext-disabled:active {
+.saltPillNext-disabled.saltPillNext-active,
```

Comment:
I don't think the styles are needed? Pill isn't focusable when disabled so can never be active when disabled